pylint/test/functional/mutable_sequence_modified_in_loop.txt

```diff
@@ -0,0 +1,5 @@
+mutable-sequence-modified-in-loop:43::"The mutable (items) sequence was modified inside the loop"
```

Comment:
What do you think about `The mutable sequence (items) was modified inside the loop`? For me it seems like sentence flow is nicer, but I'm not native English speaker so I can't be sure.